users/vieting/jobs/fairseq_finetune.py

```diff
@@ -0,0 +1,194 @@
+import soundfile
```

Comment:
I'd like to keep the filename a bit more generic so that we could possibly add other fairseq (but not necessarily fine-tinung) related jobs here. So `fairseq.py` seems reasonable to me.